src/types/Session.ts

```diff
@@ -0,0 +1,63 @@
+import { DecryptCallback, DidResourceUri, EncryptCallback, SignCallback } from '@kiltprotocol/types'
+
+import { IEncryptedMessage, IEncryptedMessageV1 } from './index.js'
```

Comment:
```suggestion
import type { IEncryptedMessage, IEncryptedMessageV1 } from './index.js'
```

Reply:
add in: [3299408](https://github.com/KILTprotocol/kilt-extension-api/pull/24/commits/329940803b8b24bcb5590bca479c6cb4d1e1011e)